src/server/routers/git.ts

```diff
@@ -10,6 +10,11 @@ import {
 import { logger } from '../../utils/logger'
 import { procedure, router } from '../trpc'
 
+const config = {
```

Comment:
Is this used?

Reply:
:DOH: remnants of my in-progress emu work